state.go

```diff
@@ -336,6 +349,7 @@ func (m *Memberlist) probeNode(node *nodeState) {
 		m.logger.Printf("[DEBUG] memberlist: Failed ping: %v (timeout reached)", node.Name)
 	}
 
+TCPCONTINUE:
```

Comment:
This label jumps over the code that waits for the response to the UDP request, since we know when it didn't happen when using TCP.